fw/vhost.c

```diff
@@ -2043,26 +2064,44 @@ tfw_cfgop_frang_glob_set_bool(TfwCfgSpec *cs, TfwCfgEntry *ce)
 	 * 'frang_limits' section may appear multiple times to modify defaults
 	 * values for future 'frang_limits' directives.
 	 */
-	if (ce->dflt_value && *(bool *)(cs->dest))
+	if (ce->dflt_value && cs->__called_cfg)
 		return 0;
 	return tfw_cfg_set_bool(cs, ce);
 }
 
 static int
 tfw_cfgop_frang_glob_set_int(TfwCfgSpec *cs, TfwCfgEntry *ce)
 {
-	if (ce->dflt_value && *(unsigned int *)(cs->dest))
+	if (ce->dflt_value && cs->__called_cfg)
```

Comment:
It seems we can't use `__called_cfg` for this purpose. This value is not zeroed after fail reload. https://github.com/tempesta-tech/tempesta-test/pull/598/commits/3683828fff2a921e2f6ee6c2666192dcf78614fc look last test It doesn't pass

Reply:
Initially, I used a separate variable, but found that it almost always changes in parallel with __called_cfg.
I fixed this error.

Reply:
Seems we still have error here for following case:
RC1:
```
http_limits {
directive-correct;
directive-incorrect;
}
```
RC2:
```
http_limits {
another-new-directive;
}
```
If during the first reconfiguration using RC1 config, we fail on `directive-incorrect` after calling handler for `directive-correct`, then on the second reconfiguration using RC2, default value for `directive-correct` will not be set.